cores/arduino/analog.cpp

```diff
@@ -116,7 +118,7 @@ void set_dac_value(PinName pinname, uint16_t value)
 #if defined(GD32F30x) || (defined(GD32F1x0) && defined(GD32F170_190)) || defined(GD32E50X)
         dac_data_set(dac_periph, DAC_ALIGN_12B_R, value);
 #elif defined(GD32F10x) && !defined(GD32F170_190)
-        dac0_data_set(DAC_ALIGN_12B_R, value);
+     //   dac0_data_set(DAC_ALIGN_12B_R, value);
```

Comment:
Would you just want to remove the F10x here?

Reply:
fixed